website/docs/cloud-docs/run/ui.mdx

```diff
@@ -65,7 +65,7 @@ To help you review proposed changes, Terraform Cloud can automatically run [spec
 
 ### Viewing Pull Request Plans
 
-Speculative plans don't appear in a workspace's list of normal runs. Instead, Terraform Cloud adds a link to the run in the pull request itself, along with an indicator of the run's status.
+Speculative plans can be viewed in a workspace's list of normal runs. Additionally, Terraform Cloud adds a link to the run in the pull request itself, along with an indicator of the run's status.
```

Comment:
```suggestion
You can view speculative plans in a workspace's list of normal runs. Additionally, Terraform Cloud adds a link to the run in the pull request itself, along with an indicator of the run's status.
```

Suggest changing "can be viewed" to "You can view" to avoid a passive voice style issue.